cpp/kiss_icp/3rdparty/sophus/sophus.cmake

```diff
@@ -31,11 +31,8 @@ ExternalProject_Add(
   CONFIGURE_COMMAND ""
   BUILD_COMMAND ""
   INSTALL_COMMAND ""
-  CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>
-             -DSOPHUS_USE_BASIC_LOGGING=ON
-             -DBUILD_SOPHUS_EXAMPLES=OFF
-             -DBUILD_SOPHUS_TESTS=OFF
-             -DCMAKE_BUILD_TYPE=Release)
+  CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR> -DSOPHUS_USE_BASIC_LOGGING=ON -DBUILD_SOPHUS_EXAMPLES=OFF
```

Comment:
This and other CMake related changes are due to cmake-format.

Reply:
Fine 👌